Python/ast.c

```diff
@@ -698,6 +698,13 @@ ast_error(struct compiling *c, const node *n, const char *errmsg, ...)
    small_stmt elements is returned.
 */
 
+static string
+new_type_comment(const char *s, struct compiling *c)
```

Comment:
The argument `c` is not used. Why is ti needed?

Reply:
No idea, I presume a legacy from an ancient typed_ast version (it would have been needed to raise exceptions).

Reply:
FWIW, this was actually a clue in the refleak mystery. The `c` argument was needed to pass to the call to `PyArena_AddPyObject()`. That call was missing, causing the refleak. ;-)